context/app/static/js/components/Routes/Routes.jsx

```diff
@@ -143,14 +147,38 @@ function Routes(props) {
     );
   }
 
-  if (urlPath.startsWith('/publication')) {
+  if (urlPath.startsWith('/publication/')) {
     return (
       <Route>
         <Publication title={title} vitData={vitessce_conf} markdown={markdown} />
       </Route>
     );
   }
 
+  if (urlPath === '/organ') {
+    return (
+      <Route>
+        <Organs organs={organs} />
+      </Route>
+    );
+  }
+
+  if (urlPath.startsWith('/organ/')) {
+    return (
+      <Route>
+        <Organ organ={organ} />
+      </Route>
+    );
+  }
+
+  if (urlPath.startsWith('/publication')) {
+    return (
+      <Route>
+        <Organ organ={organ} />
+      </Route>
+    );
+  }
+
```

Comment:
Could these be combined or will they separate down the road?

Reply:
Bad copy and paste on my part: Thanks for catching... and I should add some tests.